src/backend/main.ts

```diff
@@ -1474,6 +1474,10 @@ ipcMain.handle('syncGOGSaves', async (event, gogSaves, appName, arg) =>
   getGame(appName, 'gog').syncSaves(arg, '', gogSaves)
 )
 
+ipcMain.handle('readGOGGameInfoFile', async (event, appName: string) =>
+  GOGLibrary.get().readInfoFile(appName)
+)
```

Comment:
This is purely my opinion, but the Frontend just being able to read the whole .info file sounds a little... extreme? I believe we've also had some talks about doing more logic in the Backend in the past

Factoring this in a way that the Frontend just asks for the launch options and the Backend then does that work (reading out the file, filtering out tasks) might be *cleaner*

Reply:
I was thinking about it too, and wanted to hear your opinion on that. I'll move it to backend then